src/it/scala/org/zalando/spearheads/innkeeper/dao/PathsPostgresRepoSpec.scala

```diff
@@ -1,12 +1,15 @@
 package org.zalando.spearheads.innkeeper.dao
 
 import java.time.LocalDateTime
+
```

Comment:
my guess is that the automatic formatter adds them to separate JDK/Scala imports from library imports